### PR TITLE
Bnh 163 match page styling

### PR DESCRIPTION
**Pull request by TomHanmer29**
Styled match page to look like shiv's designs
image
image
image

### CODE REVIEW

File: web/Views/Admin/_MatchListTable.cshtml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        return preference == true ? "" : "font-orange";
+    }
+
+    static string GetPreferenceIcon(bool? preference)
+    {
+        return preference == true ? "bi-check-lg" : "bi-x-lg";
+    }
+}
+
+@if (Model.TenantList!.Count == 0)
```
Comment (oskarwilliams378) [inline web/Views/Admin/_MatchListTable.cshtml:16]: `!` not needed :)

File: web/Views/Admin/_MatchListTable.cshtml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+{
+    <h2 class="font-h3">There are no tenants which match these search criteria.</h2>
+    return;
```
Comment (oskarwilliams378) [inline web/Views/Admin/_MatchListTable.cshtml:19]: No return needed either :)

File: web/Views/Admin/_MatchListTable.cshtml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            <th scope="col">Email</th>
+            <th scope="col">Phone</th>
+            <th scope="col">Postcode</th>
+            <th scope="col">Type</th>
+            <th scope="col">Criteria</th>
+            <th scope="col">Match</th>
+        </tr>
+        </thead>
+        <tbody>
+        @foreach (var tenant in Model.TenantList!)
```
Comment (oskarwilliams378) [inline web/Views/Admin/_MatchListTable.cshtml:36]: Can remove the ! here :)

File: web/Views/Admin/_MatchListTable.cshtml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                        On housing benefits
+                    </div>
+                    <div class="@GetPreferenceFont(tenant.ETT)">
+                        <i class="bi me-1 @GetPreferenceIcon(tenant.ETT)"></i>
+                        In EET (Employment, Education, or Training)
+                    </div>
+                </td>
+                <td>
+                    @using (Html.BeginForm("SendMatchLinkEmail", "Admin", new
+                        {
```
Comment (oskarwilliams378) [inline web/Views/Admin/_MatchListTable.cshtml:68]: Weird indentation here :)

File: web/Views/Property/ViewProperty.cshtml [hunk 5]
```diff
@@ -26,6 +26,13 @@
     </div>
 
     <div class="col-md d-flex flex-column pb-2">
+        <div class="d-flex">
+            <h3 class="font-h3 d-inline-block">@(User.IsInRole("Landlord") || User.IsInRole("Admin") ? "Tenant preferences" : "Suitable for")</h3>
+            @if (User.IsInRole("Landlord") || User.IsInRole("Admin"))
```
Comment (oskarwilliams378) [inline web/Views/Property/ViewProperty.cshtml:31]: I've seen this in a lot of different places, and I think this is an example where adding an extension to the ClaimsPrincipal class (what User is) that is something like `IsLandlordOrAdmin` :)

**Review by oskarwilliams378 — CHANGES_REQUESTED** [review]

File: web/Views/Admin/_MatchListTable.cshtml (hunk 2 above)
Comment (TomHanmer29) [reply web/Views/Admin/_MatchListTable.cshtml:19]: I think we want the return here so that if there are no tenants it doesn't display the table headers.

Comment (oskarwilliams378) [reply web/Views/Admin/_MatchListTable.cshtml:19]: Sure

File: web/CustomExtensions/UserExtensions.cs [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+namespace BricksAndHearts.CustomExtensions
+{
+    public static class UserExtensions
+    {
+        public static bool IsLandlordOrAdmin(this ClaimsPrincipal user)
+        {
+            return user.IsInRole("Landlord") || user.IsInRole("Admin");
+        }
+    }   
+}
```
Comment (oskarwilliams378) [inline web/CustomExtensions/UserExtensions.cs:12]: We can use file scoping as we do elsewhere :) 
```suggestionusing System.Security.Claims;
namespace BricksAndHearts.CustomExtensions;

public static class UserExtensions
{
    public static bool IsLandlordOrAdmin(this ClaimsPrincipal user)
    {
        return user.IsInRole("Landlord") || user.IsInRole("Admin");
    }
}

```

**Review by oskarwilliams378 — APPROVED** [review]

**Review by SleepyUser — APPROVED** [review]

**Pull request merged**